crates/noseyparker-cli/src/args.rs

```diff
@@ -346,6 +348,12 @@ pub struct GitHubRepoSpecifiers {
     /// This option can be repeated.
     #[arg(long, visible_alias = "org")]
     pub organization: Vec<String>,
+
+    /// Clone and scan accessible repositories in the GitHub instance specified with the custom API url
+    ///
+    /// Requires setting up the `github_api_url` option.
+    #[arg(long, visible_alias = "scan-instance")]
+    pub scan_github_enterprise_instance: bool,
```

Comment:
It looks like the `/organizations` REST API isn't GHES-specific, but also is available on regular GitHub?

- https://docs.github.com/en/enterprise-server@3.11/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations
- https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations

If this is indeed the case, I'd prefer to make the CLI option more generic, so that it's not GHES-specific. (Though in practice, trying to scan _every_ public GitHub organization is impractical, as there are nearly 160M organizations there today.)

Let's call this option `all-organizations` instead of `scan-instance` / `scan-github-enterprise-instance`.

Reply:
Changed there and in the other instance as well.